src/stactools/landsat/commands.py

```diff
@@ -1,69 +1,169 @@
 import os
 
 import click
-import pystac
+from click import Command, Group
+from pystac import CatalogType, Item
 
-from stactools.landsat.stac import create_stac_item
+from stactools.landsat.stac import create_collection, create_stac_item
 from stactools.landsat.utils import transform_stac_to_stac
 
 
-def create_landsat_command(cli):
-    """Creates a command group for working
-    with Landsat metadata from USGS' Collection 2
+def create_landsat_command(cli: Group) -> Command:
+    """Creates a command group for commands working with the USGS' Landsat
+    Collection 2 metadata.
     """
 
     @cli.group(
         'landsat',
         short_help=("Commands for working with Landsat Collection 2 metadata.")
     )
-    def landsat():
+    def landsat() -> None:
         pass
 
     @landsat.command(
         "create-item",
-        short_help="Create a STAC item from collection 2 scene metadata.")
+        short_help="Creates a STAC Item from Collection 2 scene metadata.")
     @click.option("--level",
                   type=click.Choice(['level-1', 'level-2'],
                                     case_sensitive=False),
                   default="level-2",
                   show_default=True,
-                  help="Product level to process")
-    @click.option("--mtl", required=True, help="HREF to an MTL file.")
-    @click.option("--output",
+                  help="Product level to process. Deprecated.")
+    @click.option("-m",
+                  "--mtl",
+                  required=True,
+                  help="HREF to the source MTL metadata xml file.")
+    @click.option("-o",
+                  "--output",
                   required=True,
                   help="HREF of directory in which to write the item.")
-    @click.option("--usgs_geometry",
+    @click.option("-u",
+                  "--usgs_geometry",
                   default=False,
                   show_default=True,
                   help="Use USGS STAC Item geometry")
-    def create_item_cmd(level: str, mtl: str, output: str,
-                        usgs_geometry: bool):
-        """Creates a STAC Item for a Landsat 8 C2 Level-2 scene's products.
+    @click.option("-l",
+                  "--legacy_l8",
+                  default=True,
+                  show_default=True,
+                  help="Create deprecated Landsat 8 STAC Item")
```

Comment:
Same as above, this could be defined as a flag: https://click.palletsprojects.com/en/8.0.x/options/#boolean-flags.

Reply:
Sounds good. Done.